src/main/java/me/botsko/prism/actions/ItemStackAction.java

```diff
@@ -39,26 +40,36 @@
 import org.bukkit.inventory.meta.ItemMeta;
 import org.bukkit.inventory.meta.LeatherArmorMeta;
 import org.bukkit.inventory.meta.SkullMeta;
+import org.jetbrains.annotations.Nullable;
 
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Locale;
 import java.util.Map;
-import java.util.Map.Entry;
-import java.util.Objects;
 
 public class ItemStackAction extends GenericAction {
 
     protected ItemStack item;
-    private ItemStackActionData actionData;
-    private Map<Enchantment, Integer> enchantments;
+    private ItemStackSerializer actionData;
+
     /**
      * Holds durability if no actionData yet exists.
      */
     private short tempDurability = -1;
 
+    private static ItemStackSerializer createItemStackActionData(ItemStack item, @Nullable  Map<Enchantment, Integer> enchantments) {
```

Comment:
Codacy found an issue: [Line is longer than 120 characters (found 133).](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=6368169)